app/helpers/countries_helper.rb

```diff
@@ -1,29 +1,41 @@
 module CountriesHelper
-  def if_country_get_flag yml_key
-    country = I18n.exists?("countries.#{yml_key}.image_flag")
-    flag = FileTest.exist?("app/assets/images/#{t("countries.#{yml_key}.image_flag")}")
+  def if_country_get_flag(country_name)
+    name = country_name.downcase.gsub(' ', '-')
+    flag = FileTest.exist?("app/assets/images/flags/#{name}.svg")
 
-    country && flag ? image_tag(I18n.t("countries.#{yml_key}.image_flag"), alt: '', class: 'header__icon') : nil
+    flag ? image_tag("flags/#{name}.svg", alt: '', class: 'header__icon') : nil
   end
 
   def nav_tertiary_countries
     I18n.t('countries.nav_sticky').to_json
   end
 
   def dataset_status(dataset)
-    dataset_status = @country.occurrences[dataset[:id]]
+    @country.occurrences[dataset[:id]]
   end
 
   def dataset_status_title(dataset)
-    dataset_status_title = I18n.t("shared.proportion_protected.title_#{dataset_status(dataset)}")
+    status = dataset_status(dataset) == 'present-but-unknown' ? 'data_deficient' : dataset_status(dataset)
+
+    I18n.t("shared.proportion_protected.title_#{status}")
   end
 
   def icon_class(dataset)
-    icon_class = get_habitat_icon_class(dataset[:id], dataset_status(dataset))
+    status = dataset_status(dataset) == 'present-but-unknown' ? 'present' : dataset_status(dataset)
+
+    get_habitat_icon_class(dataset[:id], status)
   end
 
   def protected_percentage(dataset)
-    protected_percentage = @country.protection_stats[dataset[:id]] ? @country.protection_stats[dataset[:id]]['protected_percentage'].round(2) : 0
+    if @country.protection_stats[dataset[:id]] 
```

Comment:
Likewise, here I would take `@country.protection_stats[dataset[:id]]` into a variable, but only a small thing. In this instance perhaps also a little easier to read.